core/services/eth/errors.go

```diff
@@ -114,19 +131,24 @@ func NewSendError(e error) *SendError {
 }
 
 // Geth/parity returns these errors if the transaction failed in such a way that:
-// 1. It can NEVER be included into a block
-// 2. Resending the transaction even with higher gas price will never change that outcome
+// 1. It will never be included into a block as a result of this send
+// 2. Resending the transaction at a different gas price will never change the outcome
 func isFatalSendError(err error) bool {
 	if err == nil {
 		return false
 	}
-	switch err.Error() {
+	s := err.Error()
+	return isGethFatal(s) || isParityFatal(s)
+}
+
+func isGethFatal(s string) bool {
+	switch s {
```

Comment:
nit: could make this the same format as the isParityFatal check i.e. vars joined by ||

Reply:
It's done like this because these are exact string matches - whereas with parity we _must_ regexp match.

Do you feel strongly about this?

Reply:
nope